src/functions2.test.js

```diff
@@ -0,0 +1,25 @@
+import { checkF, updateItem, clearF } from './functions2.js';
+
+jest.mock('./functions2.js');
+
+describe('Test the check, update and clear functions', () => {
+  test('Check function test with true item', () => {
+    const arr = [{ completed: true }];
+    const e = 0;
+    checkF(e, 0, arr);
+    expect(arr[0].completed).toBe(false);
+  });
+
+  test('Check update item function', () => {
+    const arr = [{ description: 'Hello', completed: true }];
+    const e = 'bye';
+    updateItem(arr, 0, e);
+    expect(arr[0].description).toBe('bye');
+  });
+
+  test('Check update item function', () => {
+    let arr = [{ completed: true }, { completed: true }, { completed: false }];
+    arr = clearF(arr);
+    expect(arr.length).toBe(1);
+  });
+});
```

Comment:
- Good job on the work you guys have done 👏 👏 . When testing, it is important that we test thoroughly to ensure that our functions pass most of the test cases. Please I'd suggest that you had more tests to your project 🙏 🙏 

- Also it is important to include all properties contained in the task object, `description`, `completed`, `index` for each and every test. Please try to fix this 👍 👍